utSuiteNavigator.py

```diff
@@ -242,7 +242,7 @@ class UTSuiteNavigatorClass:
     """
     Navigates through the UTcore menu system, trigger the execution of test cases, and collect results.
     """
-    def __init__(self, config:str, startKey:str, session:consoleInterface, log:logModule=None):
+    def __init__(self, config:str, startKey:str, session:consoleInterface,  devicePath:str = "/tmp/", log:logModule=None):
```

Comment:
not sure `devicePath` is the right name for it... What about `workSpacePath`?

Reply:
done